src/binary/ibc-relayer/commands/start.ts

```diff
@@ -108,14 +108,12 @@ type Options = {
 } & LoopOptions;
 
 // some defaults for looping
-const defaultOptions: LoopOptions = {
+export const defaults = {
```

Comment:
Why have we lost the type here?

Reply:
It just doesn't make sense to me to have this type there.

`LoopOptions` type contains
```
  once: boolean;
```
it can be only true/false (guaranteed by commander) and we don't need to define its default.

Having the type here doesn't give us any benefit because it's still the developer's responsibility to attach those values in the right place.

Also, I want to be consistent with other defaults definition.